src/decision/test/gps-decision-test.cpp

```diff
@@ -1,13 +1,41 @@
 /*
- * Created By: Gareth Ellis
- * Created On: July 22, 2016
+ * Created By: Yu Chen
+ * Created On: Oct 15th, 2016
  * Description: Tests for GpsDecision
  */
 
-#include <LidarDecision.h>
+#include <GpsDecision.h>
 #include <gtest/gtest.h>
+#include <geometry_msgs/Point.h>
 
+TEST(GpsDecision,distance) {
+    geometry_msgs::Point mypoint;
+    mypoint.x = 6;
+    mypoint.y = 3;
+    mypoint.z = 0;
+    double expected=sqrt(pow(mypoint.x,2)+pow(mypoint.y,2)+pow(mypoint.z,2));
+    geometry_msgs::Point::ConstPtr mypointer(new geometry_msgs::Point(mypoint));
+    EXPECT_DOUBLE_EQ(expected, GpsDecision::distance(mypointer));
+}
+
+TEST(GpsDecision,desiredAngle) {
+    #define PI acos(-1.0)
```

Comment:
Don't think you need this, should just me able to use `M_PI` as you have elsewhere in the code (I think)